source/agora/node/data/EncryptionKey.d

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+
+    Defines the EncryptionKey.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.data.EncryptionKey;
+
+import agora.common.Hash;
+import agora.common.Types : Height;
+
+public struct EncryptionKey
+{
+    public string app;
+
+    public Height height;
+
+    public Hash value;
+}
+
+unittest
+{
+    import agora.common.Serializer;
+    testSymmetry!EncryptionKey();
+}
```

Comment:
Since this is only used inside of the QR code facility and not shared with the rest of the code, it should be in that module. Also we don't need  the symmetry test (although they don't hurt) because we never serialize it.

Reply:
I will move these structures into `QRCodeInterface.d`